tasks/2017/create-visual-art.md

```diff
@@ -0,0 +1,164 @@
+- using a vector art editor such as [Vectr](https://vectr.com/)
+- using a pixel art editor such as [Piskel](https://www.piskelapp.com/)
+- using an image editor such as [GIMP](https://www.gimp.org/)
+
+Choose a tool that works best for you!
+
+## Task Descriptions
+
+There are three types of tasks in this category, each corresponding to one of
+the Task Types below.
+
+### Task Type A: Create user avatars
+
+Zulip would like to provide a library of cute custom avatars that users can
+choose from when creating an account. Our non-custom avatars are automatically
+generated [identicons](https://en.wikipedia.org/wiki/Identicon). In this task,
+you'll create a set of user avatars with a shared theme.
+
+Some tips on making your avatars inclusive and diverse:
+- avoiding humanoid avatars is usually a good idea
+- users often like cute animals or creatures
+- well designed initials are always welcome - if you're creating text-only
+  avatars, create a set for all 24 letters
+- smiley faces and custom emoji are used quite often
+- aliens and monsters allow for a lot of creativity
+- using Zulip brand colors and logo in creative ways can be a base for avatars
+- everyday objects, especially computer-related, might be interesting
+
+If you want to read more on inclusive avatars, here's a [great article](https://uxdesign.cc/design-avatars-that-make-sense-and-be-more-inclusive-in-the-process-d4dd6a486ea6).
+
+* Create a directory `user-avatars/<username>`, where `<username>` is
+  your GitHub username.
+
+* Design 4-5 avatar images that fit together thematically, e.g. animals,     
+  monsters, smiley faces. They should look good at 50x50px size. It’s OK for
+  the first versions of these to be rough versions; it makes sense to defer
+  spending a lot of time making them really good until you’ve gotten feedback
+  on which design directions others like.
+
+* Don't use others' work for your avatars - do not submit images from the
+  Internet. Create the avatars on your own.
+
+* Add the images to the directory above, either as SVG files or two PNG images  
+  for each avatar, at 50x50 and 300x300 pixel.
+
+* Add a commit with these files, with commit message "visual art: Add
+  a/an <theme> avatar set.", where `<theme>` is the leading theme of your
+  avatar set, e.g. cute monsters.
+
+* Submit a pull request to the `zulip/zulip-gci-submissions` repository, with
+  title "Visual Art: Create a/an <theme> avatar set." Include a link to the
+  pull request when you submit your task on the GCI website.
+
+*Completion criteria:*
+* Student submitted 4-5 distinct avatars (24 if avatars are text-only) either
+  in SVG format or as 2 PNGs (50x50px and 300x300px) per avatar.
+* The avatars share a theme, e.g. it's a set of animals, monsters or smiley
+faces.
+* The avatars' distinct features are clear in the 50x50px image.
+* The avatars are student's own work, not images downloaded from the Internet.
```

Comment:
What do you think about changing this to suggest, rather than making a set of avatars, making a proposed procedurally-generated avatar set for Zulip that we could use to replace Gravatar?

@rishig FYI, since we were just talking about that as something Zulip needs.

Reply:
I think it would be a fun exercise, although it depends on how students want to approach visual art - if they're really into design, creating a set from scratch might be more fun for them :)

Reply:
I'm going to add a variant to it - if a student prefers to do a generative art set, that would be great!